crates/ext_iceberg/src/catalog/rest.rs

```diff
@@ -0,0 +1,257 @@
+use std::sync::Arc;
+
+use futures::{Stream, stream};
+use glaredb_core::catalog::create::{
+    CreateAggregateFunctionInfo,
+    CreateScalarFunctionInfo,
+    CreateSchemaInfo,
+    CreateTableFunctionInfo,
+    CreateTableInfo,
+    CreateViewInfo,
+};
+use glaredb_core::catalog::drop::DropInfo;
+use glaredb_core::catalog::entry::{CatalogEntry, CatalogEntryType};
+use glaredb_core::catalog::{Catalog, Schema};
+use glaredb_core::execution::operators::PlannedOperator;
+use glaredb_core::execution::planner::OperatorIdGen;
+use glaredb_core::storage::storage_manager::{StorageManager, StorageTableId};
+use glaredb_error::{DbError, Result};
+
+#[derive(Debug)]
+pub struct RESTCatalog {
```

Comment:
```suggestion
pub struct RestCatalog {
```

Same with RestSchema